src/components/imageDownloader/imageDownloader.js

```diff
@@ -31,19 +31,24 @@ import template from './imageDownloader.template.html';
     let browsableImageStartIndex = 0;
     let browsableImageType = 'Primary';
     let selectedProvider;
+    let browsableParentId;
 
-    function getBaseRemoteOptions() {
+    function getBaseRemoteOptions(page) {
         const options = {};
 
-        options.itemId = currentItemId;
+        if (page.querySelector('#chkShowParentImages').checked && browsableParentId != null) {
```

Comment:
The `browsableParentId != null` comparison seems a bit odd here since `browsableParentId` will likely be undefined rather than null in most cases. It works but only because it is not using the strict equality comparison. I think just using `&& browsableParentId` as a check would be better to avoid this ambiguity.